features/fixtures/handled/browserify/src/b.js

```diff
@@ -0,0 +1,17 @@
+var bugsnag = require('bugsnag-js')
+var config = require('./lib/config')
+
+var bugsnagClient = bugsnag(config)
+
+try {
+  foo.bar()
+} catch (e) {
+  bugsnagClient.notify(e, {
+    beforeSend: function () {
+      setTimeout(function () {
+        var el = document.getElementById('bugsnag-test-state')
+        el.textContent = el.innerText = 'DONE'
+      }, 5000)
```

Comment:
This `beforeSend` function appears to be the same in a few files. Is there any way of extracting this code to one location?

Reply:
Having it duplicated means that each e2e test fixture is self-contained, making it easier to follow and and reduces the chance that a slight change in a helper library could break/invalidate the test result.